cardano_node_tests/tests/tests_conway/test_pparam_update.py

```diff
@@ -1194,7 +1196,7 @@ def _check_proposed_pparams(
         )
 
         def _check_state(state: dict):
-            pparams = state.get("curPParams") or state.get("currentPParams") or {}
+            pparams = state.get("curPParams") or state.get("currentPParams") or state or {}
             clusterlib_utils.check_updated_params(
                 update_proposals=fin_update_proposals, protocol_params=pparams
```

Comment:
We already have this `clusterlib_utils.check_updated_params` function for comparing pparams dict with update proposal. We can re-use it also for checking future pparams query output.